docs/messages/event.md

```diff
@@ -1,6 +1,11 @@
+[**UDMI**](../../) / [**Docs**](../) / [**Messages**](./)
+/ [Events](#)
+
 # Events
 
 Events can be one of:
-- Pointset (telemetry)
-- System
-- Discovery
\ No newline at end of file
+- [Pointset (telemetry)](pointset.md#telemetry) ([_🧬schema_](../../gencode/docs/event_pointset.html))
```

Comment:
It was difficult to differentiate the link to the docs from the link to the schema in a few places (e.g. in this instance), because it was blue text next to blue text. So I picked a unicode emoji which looked like schema-like and used it infant of every link to the schema including different blocks (e.g. [🧬Pointset Status](../../gencode/docs/state.html#pointset_points_pattern1_status) ), so when a reader sees that they'll know it takes them to the schema.